ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -368,22 +386,40 @@ export default {
         if (matches.target !== data.me)
           return;
 
-        // Formless double tankbuster mechanic.
-        if (id === '00DA') {
-          if (data.role === 'tank')
-            return { alertText: output.formlessBusterAndSwap() };
-          // Not that you personally can do anything about it, but maybe this
-          // is your cue to yell on voice comms for cover.
-          return { alarmText: output.formlessBusterOnYOU() };
-        }
-
         // Titan Mechanics (double blue handled above)
         if (id === '00BB' && !data.seenFirstBombs)
           return { alarmText: output.titanBlueSingular() };
         if (id === '00B9')
           return { alertText: output.titanYellowSpread() };
         if (id === '00BA')
           return { infoText: output.titanOrangeStack() };
+      },
+    },
+    {
+      // Headmarkers are randomized, so use a generic headMarker regex with no criteria.
+      id: 'E12S Promise Headmarker - Chiseled Sculpture',
```

Comment:
```suggestion
      id: 'E12S Promise Chiseled Sculpture',
```

Reply:
No conflict on this one. If it still makes sense to rename given the others, I'm fine with that.